### ISSUE TITLE
Incorrect Apache config generated for TLS challenge on IPv6 host

### ISSUE THREAD

**Issue by jsha** [letsencrypt/letsencrypt#2613]
See https://community.letsencrypt.org/t/please-add-ipv6-support-to-the-client/12006

```
After doing some debugging, it turned out that the client is not supporting IPv6. It writes a file called /etc/apache2/le_tls_sni_01_cert_challenge.conf with contains the following lines:

<VirtualHost 46.4.113.142:443 [2a01:443>
...
<VirtualHost [2a01:443>
```

**Comment by bmw** [comment]
This should be resolved with #1143 which has a PR and adds IPv6 support to the Apache plugin.

If for some reason this doesn't land, we should at least detect that a vhost uses IPv6 and refuse to use it until it is supported by the plugin.

**Comment by pde** [comment]
Should have been closed by #2690 / #2478 I believe?
